app/controllers/submissions_controller.rb

```diff
@@ -578,6 +580,31 @@ def download_groupings_files
   end
 
   ##
+  # Check the status of collection for all groupings
+  ##
+  def check_collect_status
+    grouping_ids = params[:groupings]
+
+    ## if there is no grouping, render a message
+    if grouping_ids.blank?
+      render text: t('student.submission.no_groupings_available')
+      return
+    end
+
+    groupings = Grouping.where(id: grouping_ids)
+      .includes(:group,
+                current_submission_used: {
+                  submission_files: {
+                    submission: { grouping: :group }
+                  }
+                })
+
+    ## check collection is completed for all groupings
+    all_groupings_collected = groupings.all?{ |g| g.is_collected? }
```

Comment:
Space missing to the left of {.<br>Pass `&:is_collected?` as an argument to `all?` instead of a block.